### PR TITLE
404 page

### PR DESCRIPTION
**Pull request by stanleypliu**
**Ticket**

https://unep-wcmc.codebasehq.com/projects/pp-refresh/tickets/216

404 page created with news articles, resource cards and thematic carousel all on one page, plus a redirect back to the homepage. 

I'd like to wait for Charlotte's feedback first though before review

### CODE REVIEW

**Comment by stanleypliu** [comment]
@Levia can be reviewed now

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -16,6 +16,14 @@ class PageNotFound < StandardError; end;
   before_action :check_for_pdf
   after_action :store_location
 
+  NO_REDIRECT = ["/users/sign_in",
+    "/users/sign_up",
+    "/users/password/new",
+    "/users/password/edit",
+    "/users/confirmation",
+    "/users/sign_out"
+  ].freeze
```
Comment (Levia) [inline app/controllers/application_controller.rb:25]: I wonder if we should get rid of these as I don't think users are still able to log in.
This is something that precedes me as well but looking at the current website those pages look a bit broken :)

Comment (stanleypliu) [reply app/controllers/application_controller.rb:25]: Didn't know it was a holdover, and yes I know :)

**Comment by Levia** [comment]
Looks good! Merging

**Pull request merged**